qutebrowser/app.py

```diff
@@ -70,7 +70,7 @@ def run(args):
         sys.exit(usertypes.Exit.ok)
 
     if args.temp_basedir:
-        args.basedir = tempfile.mkdtemp()
+        args.basedir = tempfile.mkdtemp(prefix='qutebrowser-prefix-')
```

Comment:
As discussed in IRC (just so it doesn't get lost):

This probably should be `-basedir-`, not `-prefix-`